identity/app/views/advertsManager.scala.html

```diff
@@ -19,8 +19,8 @@ <h2 class="form__heading">Tracking</h2>
 
         <div class="fieldset__fields">
             <p class="identity-title-explainer">Google uses personalisation to deliver better ads. Unticking this will not reduce the number of ads in The Guardian but it will make them less relevant to your interests</p>
-            <div class="js-manage-account__ad-prefs is-hidden">
-                <span>Loading</span>
+            <div class="js-manage-account__ad-prefs">
+                <span>Loading…</span>
```

Comment:
Unicode ellipses? Not sure if that's a good idea

Reply:
WIll replace these